edk2toollib/windows/capsule/inf_generator2.py

```diff
@@ -331,7 +331,7 @@ def __str__(self) -> str:
         outstr += Files
         outstr += textwrap.dedent("""
             [DestinationDirs]
-            DefaultDestDir = %DIRID_WINDOWS%,Firmware ; %SystemRoot%\\Firmware
+            DefaultDestDir = 13
```

Comment:
I believe line 240 needs to be updated as well.

https://learn.microsoft.com/en-us/windows-hardware/drivers/bringup/authoring-an-update-driver-package
HKR,,FirmwareFilename,,%13%\firmware.bin
; Prior to Windows 10 1803, the above should instead be:
; HKR,,FirmwareFilename,,{6bd4efb9-23cc-4b4a-ac37-016517413e9a}\firmware.bin

Reply:
Ah yes. Thank you; I will update now.